components/harvest/sources/new-timeentry-entry/new-timeentry-entry.mjs

```diff
@@ -0,0 +1,45 @@
+import harvest from "../../harvest.app.mjs";
+
+export default {
+  key: "harvest-new-timeentry-entry",
+  name: "New Time Entry",
+  description: "Emit new notifications when a new time entry is created",
+  version: "0.0.1",
+  type: "source",
+  props: {
+    harvest,
+    timer: {
+      label: "Polling interval",
+      description: "Pipedream will poll Harvest API on this schedule",
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+    db: "$.service.db",
+  },
+  dedupe: "unique",
+  async run() {
+    const data = [];
+    let lastDateChecked = this.harvest.getLastDateChecked(this.db);
+    if (!lastDateChecked) {
+      lastDateChecked = new Date().toISOString();
+      this.harvest.setLastDateChecked(this.db, lastDateChecked);
+    }
+    const time_entries = await this.harvest.listTimeEntriesPaginated({
+      page: 1,
+      updatedSince: lastDateChecked,
+    });
+    for await (const time_entry of time_entries) {
```

Comment:
I think don't need this `await` here.

Obs: I din't tested.

```suggestion
    for (const time_entry of time_entries) {
```

Reply:
Hi, await is required. Tested.